build/scripts/run_perf.ps1

```diff
@@ -20,3 +20,7 @@ Invoke-WebRequest -Uri http://dotnetci.blob.core.windows.net/roslyn-perf/cpc.zip
 [Environment]::SetEnvironmentVariable("VS150COMNTOOLS", "C:\\Program Files (x86)\\Microsoft Visual Studio\\VS15Preview\\Common7\\Tools", "Process")
 
 ./cibuild.cmd /testPerfRun /release
+if ($LASTEXITCODE -ne 0)
```

Comment:
The code above uses `-not $?` for checking error conditions.  Why the difference here?